kernel/rtlil.cc

```diff
@@ -3982,23 +4012,47 @@ RTLIL::SigSpec::SigSpec(bool bit)
 {
 	cover("kernel.rtlil.sigspec.init.bool");
 
+	packed_ = false;
+	(void)new (&this->bits_) std::vector<SigBit>();
 	width_ = 0;
 	hash_ = 0;
 	append(SigBit(bit));
 	check();
 }
 
+void RTLIL::SigSpec::switch_to_packed() const
```

Comment:
From the naming alone it would appear this does the same thing as `pack()`, I suggest we name it `switch_union_to_chunks` or similar.